erc20-price/src/lib.rs

```diff
@@ -32,7 +32,7 @@ fn map_price_for_tokens(
         prices.items.push(Erc20Price {
             block_number,
             price_usd: token_price.to_string(),
-            token_address: erc20_token.clone(),
+            token_address: Hex(erc20_token.clone()).to_string(),
```

Comment:
`hex::encode(erc20_token.clone())`

Reply:
what's the difference here? Seems like both do the same thing

Reply:
not different function-wise, just shorter and less overhead

Hex(...) creates a Hex struct first, then call to_string() method
while Hex::encode(...) is like calling a static method, thus no struct created, thus less overhead